src/components/Questionnaire/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+
+class Questionnaire extends Component {
+
+  render() {
+    const {sampleQuestionnaire} = this.props;
+    return (
+        <div className="questionnaire__wrapper">
+          <h2 className="questionnaire__item--title">Questionnaire</h2>
```

Comment:
I don't know if the class naming is very correct... this is the title of the Questionnaire section, so why put "item" in between? Could you clarify this for me? :)
On the other hand, I think it's good in lines 15 and 18! Except that I'm not sure we should use -- because it's not a modifier, right? I mean, wouldn't it be more correct to write `questionnaire__item-question`? I'm not sure at all, just asking because I always have doubts on this issue!

Reply:
@lauraferrandof thanks to you for your time and helpful input! That's right, I added the classNames prior to ulterior arrangement modifications, so now they don't make sense. 
`<h2>` tag will be questionnaire__title, since it's an element pertaining to block questionnaire. 
And for the two `<p>` tags, we could use questionnaire__question/answer respectively, since they are elements as well.